src/plugins/chart_expressions/expression_gauge/public/components/index.scss

```diff
@@ -4,7 +4,7 @@
   width: 100%;
   // the FocusTrap is adding extra divs which are making the visualization redraw twice
   // with a visible glitch. This make the chart library resilient to this extra reflow
-  overflow: auto hidden;
+  overflow: hidden;
```

Comment:
this prevents the little glitch - I accidentally removed the fix the moving to expression:

https://user-images.githubusercontent.com/4283304/146357812-7daf8b43-09da-4702-9ba8-aa78e00a350d.mp4